openapi/openapi.yaml

```diff
@@ -999,6 +976,176 @@ components:
           required:
             - data
 
+    UiCollection:
+      title: 'UiCollection'
+      description: 'Detailed info about a collection used by the ui'
+      type: object
+      properties:
+        all_versions:
+          readOnly: true
+          type: array
+          items:
+            $ref: '#/components/schemas/UiCollectionVersion'
+        community_score:
+          nullable: true
+          type: number
+        community_survey_count:
+          type: integer
+        deprecated:
+          type: boolean
+        download_count:
+          type: integer
+        id:
+          readOnly: true
+          type: integer
+        latest_version:
+          $ref: '#/components/schemas/UiCollectionVersionDetail'
```

Comment:
Collection version detail returns too much information for the list of collections. The list of collections should return a collection version object without data such as the readme, collection content and documentation.

Reply:
@newswangerd Agreed. The commit message has some comments about this, but basically at the moment the list and detail view both return  UiCollectionVersionsDetail. This needs to be finer grained to handle both.